### ISSUE TITLE
Page template: Rename "Create a custom template" to "Create a page template"

### ISSUE THREAD

**Issue by paaljoachim** [WordPress/gutenberg#31086]
I am working on adjustments to the Page template.

Clicking to create a New template brings up this modal.

![Create-a-custom-template-modal](https://user-images.githubusercontent.com/5323259/115714722-bbac5080-a377-11eb-9696-023c6c00580e.jpg)

A custom template does not explain. Does not give a location pointer to where it will be used.

![Create-a-page-template](https://user-images.githubusercontent.com/5323259/115715009-08902700-a378-11eb-8606-9e8d75ac28df.jpg)

Changing the name from custom to page gives a clear pointer/location to where the template will be used.

**Comment by jameskoster** [comment]
I believe the "custom" label is a reference to $custom.php templates that exist in classic themes. The point being that these templates can be applied not only to pages, but to posts (and potentially custom post types) as well.

We should be cognisant of the fact that we'll need to make it possible for new templates to be either $custom or specific to a particular post or page as well.

So perhaps a simpler label like "Create template" might be a bit more user-friendly.

**Comment by paaljoachim** [comment]
Good idea to just call it "Create template" as it then will embrace page/post and custom post types as well.

Updated:
![Create-template-modal](https://user-images.githubusercontent.com/5323259/115745841-1a7fc300-a394-11eb-8413-19d506848894.jpg)

**Comment by paaljoachim** [comment]
I will just let it go, and close this issue.

**Comment by youknowriad** [comment]
It says "create new template" now, so I'm closing this issue as solved.
